web/index.html

```diff
@@ -1,11 +1,21 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 	<head>
-		<meta charset="UTF-8">
+		<meta charset="UTF-8" />
 		<title>Flathub Stats</title>
 		<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.bundle.min.js" integrity="sha256-TQq84xX6vkwR0Qs1qH5ADkP+MvH0W+9E7TdHJsoIQiM=" crossorigin="anonymous"></script>
 <!--		https://flaviocopes.com/javascript-async-defer/-->
-		<script defer src="script.js"></script>
+		<script defer="" src="script.js"></script>
+		<meta property="og:title" content="Flathub Stats" />
+		<meta property="og:locale" content="en_US" />
+		<meta name="description" content="Stats of flathub.org" />
+		<meta property="og:description" content="Stats of flathub.org" />
+		<meta property="og:url" content="https://klausenbusk.github.io/flathub-stats/" />
+		<meta property="og:site_name" content="Flathub Stats" />
+		<meta name="twitter:card" content="summary" />
+		<meta property="twitter:title" content="Flathub Stats" />
```

Comment:
I think we can just add a `og:type` and then we don't need these two lines:

https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup
> If an og:type, og:title and og:description exist in the markup but twitter:card is absent, then a summary card may be rendered.